src/components/ContentOpenWith/IntegrationPortal.js

```diff
@@ -24,7 +24,15 @@ class IntegrationPortal extends PureComponent<Props> {
         super(props);
 
         const { integrationWindow }: Props = this.props;
-        this.containerElement = integrationWindow.document.createElement('div');
+        const existingContainer = integrationWindow.document.querySelector(
```

Comment:
You may want to apply an `id` to the div when it's created so you can select it more easily/safely.

Reply:
will do, that's safer